src/shader_compiler.h

```diff
@@ -50,6 +50,15 @@ class ShaderCompiler {
   std::string spv_env_;
 };
 
+// Parses the SPIR-V environment string, and returns three values
+// corresponding to Shaderc's values for shaderc_target_env,
```

Comment:
The Shaderc bit isn't really relevant.

Parses the SPIR-V environment string, and returns the corresponding
|target_env|, |target_env_version| and |spirv_version|. Returns a
failure value if the |spv_env| string is invalid.

Reply:
done